lib/server-info-route-handler.js

```diff
@@ -105,6 +106,19 @@ function formatInfo (json = {}, defaultId, type) {
   }
 }
 
+function relationshipInfo (json = {}, defaultId) {
```

Comment:
Does the `defaultId` have a default value? Or will the `defaultId` be provided by the caller?

Reply:
Reading this code, `relationshipInfo` is the function used in a `.map`.  So here, `defaultId` is the array index.  It should be renamed to `index` or `relationshipIndex` for clarity.

Reply:
This is consistent with the signatures of the `layerInfo`, `tableInfo`, and `formatInfo` functions. This naming convention keeps it consistent with the existing code.

Reply:
FeatureServer is an old code base and has many examples of anti-patterns.  New development is an opportunity to do things right and save refactoring work later.  Please make the change.